src/Model/__tests__/Model.test.ts

```diff
@@ -0,0 +1,111 @@
+import { FakeLogger } from '@adonisjs/logger/build/standalone';
+
+import { Mongodb } from '../../Mongodb';
+import { Model } from '../Model';
+
+interface IUser {
+  username: string;
+  password: string;
+}
+
+class User extends Model implements IUser {
```

Comment:
You can add tests for the IncrementalModel

Reply:
Done